src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -155,8 +155,22 @@ public Edit readEdit(JSONObject source) throws BadInstructionsException, CantGet
                 for (int i = 0; i < length; i++) {
                     JSONObject layout_info = args.optJSONObject(i);
                     ViewVisitor.LayoutRule params;
-                    params = new ViewVisitor.LayoutRule(layout_info.getInt("view_id"),
-                            layout_info.getInt("verb"), layout_info.getInt("anchor_id"));
+
+                    final Integer view_id = reconcileIdsInPath(-1, layout_info.getString("view_id"), mResourceIds);
```

Comment:
Let's change the name of "view_id" to something like "view_id_name" if we're going to interpret it as a name. We should also probably change the name of `reconcileIdsInPath` since this call isn't in the path interpretation code.